sdk/confidentialledger/confidential-ledger-rest/tests.yml

```diff
@@ -0,0 +1,13 @@
+trigger: none
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure/confidential-ledger"
+      ServiceDirectory: confidentialledger
+      EnvVars:
+        LEDGERURI: https://$(ledgerName).confidential-ledger.azure.com
+        LEDGER_IDENTITY: $(ledgerName)
+        IDENTITY_SERVICE_URL: https://identity.confidential-ledger.core.azure.com/ledgerIdentity/$(ledgerName)
+        LEDGER_NAME: $(ledgerName)
+      MaxParallel: 1
```

Comment:
What is the reason for restricting parallelism here? Is there a max quota on ledgers per subscription or something?